.env.test

```diff
@@ -88,3 +88,5 @@ MESSENGER_TRANSPORT_DSN=in-memory://
 
 WEBHOOK_BASIC_AUTH_USERNAME=testUsername
 WEBHOOK_BASIC_AUTH_PASSWORD=testPassword
+
+GENERATED_USERS_FILE_PATH=var/user-files/
```

Comment:
This is the same path you have used for docker environment, please use something like `/var/test/user-files` or similar